core/node/auth/auth_impl.go

```diff
@@ -414,7 +414,7 @@ func (ca *chainAuth) isEntitledToChannelUncached(
 
 	// For read, write and react permissions, fetch the entitlements and evaluate them locally.
 	if (args.permission == PermissionRead) || (args.permission == PermissionWrite) ||
-		(args.permission == PermissionReact) {
```

Comment:
I see you removed these here... https://github.com/river-build/river/pull/974/files